client/webserver/site/src/js/forms.ts

```diff
@@ -557,12 +562,18 @@ export class WalletConfigForm {
   /*
    * update creates the dynamic form.
    */
-  update (configOpts: ConfigOption[] | null, activeOrders: boolean) {
+  update (guideLink: string, configOpts: ConfigOption[] | null, activeOrders: boolean) {
     this.assetHasActiveOrders = activeOrders
     this.configElements = []
     this.configOpts = configOpts || []
     Doc.empty(this.dynamicOpts, this.defaultSettings, this.loadedSettings)
```

Comment:
As I [suggested before](https://github.com/decred/dcrdex/pull/2261#pullrequestreview-1359740617), please move the `walletCfgGuide` out of the `walletSettings` and into the `mainForm`. This is not the purview of the dynamic form section.

Reply:
Moved now to a separate `template` that gets embedded into **both** `mainForm` and `reconfigForm` https://github.com/decred/dcrdex/pull/2261/commits/26c65ba6e867c76f3a7e0d1c77dee5cf11ac0aa8, see also [why I couldn't apply your change as is](https://github.com/decred/dcrdex/pull/2261#issuecomment-1491302624).

But some JS code is duplicated now (to serve 2 different forms), not sure if you'd want to dedup it by introducing additional entity that combines settings+guide into single section or something like that, so leaving like that for now.

Reply:
Sorry. I see what you're saying now. Less duplication to put it in the already templated section. The new arguments to `update` stuck out to me as a smell though. You can put it back if you want, but I'm fine either way.

Reply:
I think it's fine to leave it like it currently is for now.